webapp/javascript/pages/continuous/contextMenu/AddAnnotation.menuitem.tsx

```
@@ -69,43 +57,43 @@ function AddAnnotation(props: AddAnnotationProps) {
       <Portal container={container}>
         <Popover
           anchorPoint={{ x: popoverAnchorPoint.x, y: popoverAnchorPoint.y }}
-          isModalOpen={isPopoverOpen}
+          isModalOpen
           setModalOpenStatus={setPopoverOpen}
         >
-          <PopoverHeader>Add annotation</PopoverHeader>
-          <PopoverBody>
-            <form
-              id="annotation-form"
-              name="annotation-form"
-              onSubmit={handleSubmit((d) => {
-                onCreateAnnotation(d.content);
-              })}
-            >
-              <TextField
-                {...register('content')}
-                label="Description"
-                variant="light"
-                errorMessage={errors.content?.message}
-              />
-              <TextField
-                label="Time"
-                type="text"
-                readOnly
-                value={format(
-                  getUTCdate(
-                    new Date(timestamp * 1000),
-                    timezoneToOffset(timezone)
-                  ),
-                  'yyyy-MM-dd HH:mm'
-                )}
-              />
-            </form>
-          </PopoverBody>
-          <PopoverFooter>
-            <Button type="submit" kind="secondary" form="annotation-form">
-              Save
-            </Button>
-          </PopoverFooter>
+          {isPopoverOpen ? (
```

Instead of doing ternary conditionals, prefer to fail early.
Example

```tsx
if (!isPopoverOpen) {
  return null;
}
```